src/app/lib/utilities/getUrlHelpers/getEmbedUrl/index.test.js

```diff
@@ -22,18 +22,6 @@ const testCases = [
       queryString: '',
     },
   },
-  {
-    description: `should build an AMP url for articles in test environment`,
-    expected: `https://polling.test.bbc.co.uk/ws/av-embeds/articles/${mediaId}/amp`,
-    environment: 'test',
-    before: setEnvironment,
-    embedObject: {
-      isAmp: true,
-      mediaId,
-      type: 'articles',
-      queryString: '',
-    },
-  },
   {
     description: `should build a CANONICAL url for articles in test environment with test override`,
```

Comment:
Does the Canonical reference need to stay in these tests or should I remove it for the remaining tests now that there's no longer AMP references anywhere in these files?